rpc/src/eth_rpc.rs

```diff
@@ -256,6 +256,16 @@ impl EthApiServer for KakarotEthRpc {
         Ok(block_number)
     }
 
+    async fn get_code(&self, _address: Address, _block_number: Option<BlockId>) -> Result<Bytes> {
+        let code = self
+            .starknet_client
+            .get_code(_address, _block_number)
```

Comment:
This this is calling a Kakarot Contract account, we must consider users input an EVM-format address and then we do a conversion using the kakarot address registry! Then, we use the Starknet address to fetch the code:)

Reply:
Ahhh okay you did it above in the get_code of Starknet client^^